sys/parsoid.js

```diff
@@ -593,16 +616,7 @@ PSP._getOriginalContent = function(restbase, req, revision, tid) {
 PSP._getStashedContent = function(restbase, req, etag) {
     var self = this;
     var rp = req.params;
-    if (!rp.title || !rp.revision) {
```

Comment:
This is only called from `transfromRevision`, which is only called if we have `title` and `revision`, so this is dead code.